library/include/rocwmma/internal/mapping_util_impl.hpp

```diff
@@ -75,41 +73,41 @@ namespace rocwmma
         template <uint32_t TBlockX, uint32_t TBlockY>
         __device__ constexpr inline auto WaveSpace<TBlockX, TBlockY>::localWaveCoord() -> WaveCoordT
```

Comment:
Should follow the class' aliases, should the class design change. Coord2d may be same as WaveCoordT right now, but if there is a change we don't want to edit all the instances for this class.

For example with this class, we should pass a fresh instance of WaveCoordT into waveCount